app/build.gradle

```diff
@@ -39,7 +39,9 @@ android {
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     amazonImplementation 'com.github.jellyfin:exomedia:amazon~2.5.6-SNAPSHOT'
+    amazonImplementation 'com.github.amzn:exoplayer-amazon-port:amazon-r1.5.6-SNAPSHOT'
     googleImplementation 'com.devbrackets.android:exomedia:2.5.6'
+    googleImplementation 'com.google.android.exoplayer:exoplayer:r1.5.6'
```

Comment:
Maybe we should add a comment to note why these are needed.

Reply:
Otoh, if anyone removes them, they will find out why pretty quickly :D